lib/features/details/presentation/view/movie_details_page.dart

```diff
@@ -0,0 +1,110 @@
+import 'package:flutter/material.dart';
+import 'package:myflix/features/home/presentation/widgets/movie_list_view.dart';
+
+class MovieDetailsPage extends StatefulWidget {
+  const MovieDetailsPage({super.key});
+
+  @override
+  State<MovieDetailsPage> createState() => _MovieDetailsPageState();
+}
+
+class _MovieDetailsPageState extends State<MovieDetailsPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Container(
+            height: 280,
+            width: MediaQuery.of(context).size.width,
+            decoration: const BoxDecoration(
+              image: DecorationImage(
+                image: AssetImage('asset/dune.jpeg'),
+              ),
+            ),
+          ),
+          const SizedBox(height: 10),
+          const Text(
+            'Cinema Paradiso',
+            style: TextStyle(
+                color: Colors.white, fontSize: 25, fontWeight: FontWeight.bold),
+          ),
+          const SizedBox(height: 10),
+          Row(
+            children: [
+              const Text(
+                '1988',
+                style: TextStyle(
+                  color: Colors.white,
+                  fontSize: 16,
+                ),
+              ),
+              const SizedBox(width: 10),
+              Container(
```

Comment:
There's a dot symbol you can use instead of this Container.
This is it: •
Use it in a Text widget.

And instead of a Row of Texts, use RichText widget instead